### ISSUE TITLE
Doesn't work on my project

### ISSUE THREAD

**Issue by bitboxer** [carakan/pronto-credo#6]
Hei, 

I tried to setup pronto-credo on [SignDict](https://github.com/signdict/website) together with travis and sadly it does not report errors. Not sure what I did wrong or how to debug this, but I am happy to help you with any information you might need.

What do you need to investigate this with me?

Thanks <3

**Comment by leifg** [comment]
I am encountering the same problem. This might be the same error. I was debugging for a little bit.

Can you try to remove [this line](https://github.com/carakan/pronto-credo/blob/master/lib/pronto/credo/output_parser.rb#L21)

And see what the output is? That did the trick for me (the path that is checked is an absolute path and will therefore never match the credo relative path).

I am not sure what the implications of removing this line will be.

**Comment by leifg** [comment]
Follow Up:

There is an [open PR](https://github.com/carakan/pronto-credo/pull/4) which seems to deal with this issue. Specifically [this commit](https://github.com/carakan/pronto-credo/pull/4/commits/5677a48972dab75013a3d972fb5ac166aab3b691) fixes it for me.

**Comment by bitboxer** [comment]
Yes, this fixes it for me. Would love that PR to be finished and merged 👍 . 

@carakan: If there is anything that I can do to speed that up, please ping me. I can help to fix any outstanding things.

**Comment by leifg** [comment]
For a workaround, I forked this repo and applied the fix. I got it to run via [specific_install](https://github.com/rdp/specific_install):

```shell
gem install specific_install
gem specific_install -l https://github.com/leifg/pronto-credo
```

**Comment by carakan** [comment]
sorry @bitboxer @leifg  for the delay, I pick that commit https://github.com/carakan/pronto-credo/pull/4/commits/5677a48972dab75013a3d972fb5ac166aab3b691 and will make a new version, so about of PR #4 I'm going to create a test suite to handle correctly all the cases in this new BIG change, also I'll create two ways to use pronto with credo (if is possible in my time): the traditional it will the current functionality and a new with more detailed errors added with that PR, to avoid some weird and undesirable changes.
